README.md

```diff
@@ -74,12 +74,37 @@ docker run -it --rm -v sqlite:/sqlite django-docker-template:master python manag
 
 6. Start all services locally (Postgres, Gunicorn, Traefik) using docker-compose:
 ```console
-docker compose -f docker-compose.debug.yml up
+docker compose -f docker-compose.debug.yml up -d --build
 ```
 
 Enjoy watching the lines run in the terminal 🖥️   
 And after a few seconds, open your browser at http://127.0.0.1/admin/. The first user already exists, welcome to the Django admin panel.
 
+### More development commands
+
+With the containers running, you can run manage.py commands using this format:
+
+- Run test suite
+ ```console
+ docker-compose exec django python manage.py test
+ ```
+- Make migrations
+ ```console
+ docker-compose exec django python manage.py makemigrations
```

Comment:
Could you also leave just a couple of example commands, like 'shell' and 'test', to demonstrate? This will help keep the description short and sweet. People can figure out the rest without going through every command. Let's avoid lengthy manuals :)